public/editor/stylesheets/editor.less

```diff
@@ -634,3 +634,80 @@
 .hide {
   display: none;
 }
+
+
+
+/*
```

Comment:
I think you should move this to an inline `<style>...</style>` in the editor HTML so that it renders instantly vs. blocking on the network.